pyproject.toml

```diff
@@ -131,6 +136,8 @@ select = [
     "I",   # isort
     # "SIM", # flake8-simplify
 ]
+ignore = ["E402", "I001"]
```

Comment:
why has this been switched off globally ? Please fix these code style issues rather than ignoring (especially not globally).